pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -89,6 +104,8 @@ var (
 	}
 
 	taintKeyToNodeConditionMap = map[string]v1.NodeConditionType{
+		algorithm.TaintNodeNotReady:           v1.NodeReady,
+		algorithm.TaintNodeUnreachable:        v1.NodeReady,
```

Comment:
let's ignore unreachable in this PR to keep consistent.

Reply:
Is there any different taint behavior between `NotReady` and `Unreachable`?